sdk/network/Azure.ResourceManager.Network/src/Generated/Models/ApplicationGatewaySslProfile.cs

```diff
@@ -51,7 +51,19 @@ internal ApplicationGatewaySslProfile(string id, string name, string etag, strin
         /// <summary> SSL policy of the application gateway resource. </summary>
         public ApplicationGatewaySslPolicy SslPolicy { get; set; }
         /// <summary> Client authentication configuration of the application gateway resource. </summary>
-        public ApplicationGatewayClientAuthConfiguration ClientAuthConfiguration { get; set; }
+        internal ApplicationGatewayClientAuthConfiguration ClientAuthConfiguration { get; set; }
+        /// <summary> Verify client certificate issuer name on the application gateway. </summary>
+        public bool? VerifyClientCertIssuerDN
```

Comment:
this name seems not have much context, maybe `ClientAuthConfigurationVerifyClientCertIssuerDN` is better (though it is quite long)

Reply:
boolean we simply keep the same name since they should start with a verb